### PR TITLE
perf(model): remove unecessary clone/item calls

### PR DESCRIPTION
**Pull request by ClashLuke**
The loss may be propagated a bit more slowly and some IDEs may give warnings about aux_loss being used before assignment. Other than that I don't see any issue.\
I did however not test this code and simply hope for it to work.

### CODE REVIEW

File: lightweight_gan/lightweight_gan.py [hunk 1]
```diff
@@ -782,21 +783,24 @@ def train(self):
...
-            total_disc_loss += divergence.detach().item() / self.gradient_accumulate_every
+            if i % log_step == 0:
+                total_disc_loss += divergence
 
-        self.d_loss = float(total_disc_loss)
+        del total_disc_loss
+        self.last_recon_loss = aux_loss.item()
+        if apply_gradient_penalty:
+            self.last_gp_loss = gp.item()
+        self.d_loss = float(total_disc_loss.item() / self.gradient_accumulate_every)
```
Comment (lucidrains) [inline lightweight_gan/lightweight_gan.py:803]: If the `total_disc_loss` was deleted on line 799, is it still present here?

**Comment by lucidrains** [comment]
@ClashLuke I'll take it! Thank you!
